build/gulp/sh.ts

```diff
@@ -1,28 +1,13 @@
-import { spawn } from 'child_process'
-
-const sh = (command, cb) => {
-  const [cmd, ...args] = command.split(' ')
+import { exec } from 'child_process'
 
+const sh = (cmd, cb) => {
   const options = {
     cwd: process.cwd(),
     env: process.env,
+    stdio: 'inherit',
   }
 
-  const child = spawn(cmd, args, options)
```

Comment:
let me, please, try this thing before merging. I've used to use 'spawn' quite often while working on Windows machine, and it haven't introduced any problems for me. Just to be sure where the problem is

Reply:
Sure, I introduced this PR just to unblock other developers on windows as soon as possible if there are any others.

Reply:
For context, command output from `exec` was being swallowed despite inheriting the streams.  Spawn allowed us to properly log the output of commands we were running.

If we need to go back to exec we can, but it would be helpful to preserve logging if possible.